config/faf.conf.in

```diff
@@ -30,3 +30,10 @@ CreateComponents = False
 # allowed values: fedora-bugzilla rhel-bugzilla centos-mantisb comment email url
 # or * to allow all attachments
 AcceptAttachments = fedora-bugzilla rhel-bugzilla centos-mantisbt
+
+# Determines which strategy will be used for searching known or uknown ureport's
+# and bugzilla bug's, if known is empty, then is used defaul searching
```

Comment:
"then the default is used".

What is the default? Please also mention what OS_MAJOR and OS_MINOR does instead of referencing the github issue..